pages/common/beanstalkd.md

```diff
@@ -0,0 +1,19 @@
+# beanstalkd
+
+> A simple and generic work-queue server.
+
+- Start beanstalkd, listening on port 11300:
+
+`beanstalkd`
+
+- Start beanstalkd listening on a custom port and address:
+
+`beanstalkd -l {{ip_address}} -p {{port_number}}`
```

Comment:
Specifying a different ip address is only possible when a machine has 2 ip addresses right ? Or did you mean like `localhost` and the public interface ?

Reply:
I meant to say the IP that it listens on. It defaults to `0.0.0.0`.

Reply:
Hmm .. ok.